src/FSharp.Core/array.fs

```diff
@@ -2054,6 +2056,126 @@ module Array =
 
             result
 
+        // The following two parameters were benchmarked and found to be optimal.
```

Comment:
Can benchmarks be committed as well?

Reply:
There is a linked repo attached to this PR.
I do not think those belong to F# repo, they were just guidance for picking from a variety of implementation options.